README.md

```diff
@@ -32,6 +28,10 @@ Mobly tests could run on the following platforms:
   - MacOS 10.6+
   - Windows 7+
 
+| Linux | Windows |
```

Comment:
Can we make it look like:

Platform    Build Status
Linux            [badge]
Windows      [badge]

Reply:
changed